aptos-move/framework/README.md

```diff
@@ -34,6 +34,40 @@ aptos move document --help
 
 The documentation process is also integrated into the framework building process and will be automatically triggered like other derived artifacts, via `cached-packages` or explicit release building.
 
+## Running Move tests
+
+To test our Move code while developing the Aptos Framework, run `cargo test` inside this directory:
+
+```
+cargo test
```

Comment:
`cargo test -p aptos-framework` works anywhere to run the tests as well

Reply:
Fixed.